lib/tasks/remap_tags.rake

```diff
@@ -0,0 +1,63 @@
+namespace :remap_tags do
+
+  desc "this task remaps and deletes tags. This is a one time task!"
+  task execute: :environment do
+    map_tags
+    rename_and_delete_tags
+    puts "Completed!!!"
+  end
+
+  def map_tags
+    tags = {
+            "food" => ["meals"],
+            "fellows-lagos" => ["fellows lagos", "lagos"],
+            "php" => ["pdo", "laravel", "dbal"],
+            "training" => ["cousera", "simulations"],
+            "sports" =>["football"],
+            "healthcare" => ["kbl", "bupa", "health"],
+            "javascript" => ["dom", "superagent"],
+            "react" => ["react router"],
+            "ruby" => ["rspec", "serializers"],
+            "testing" => ["faker", "mock", "stub", "unit-testing"],
+            "operations" => ["leave", "leave-request", "annual-leave"],
+            "mac" => ["osx"],
+            "nairobi-fellows" => ["brown-bag", "kenya"],
+            "databases" => ["sql", "database-relationships"],
```

Comment:
Before this line we will have to rename `database relationship` to `database-relationships`